modules/appeals_api/app/swagger/appeals_api/v1/notice_of_disagreements_controller_swagger.rb

```diff
@@ -508,14 +512,14 @@ class AppealsApi::V1::NoticeOfDisagreementsControllerSwagger
                 end
 
                 property :status do
-                  key :description, 'status description here...'
+                  key :description, ''
```

Comment:
Maybe remove the descriptions altogether if they're empty strings?

Reply:
I'm seeing this in our NOD documentation as well. Is this a placeholder?

Reply:
I'm seeing this in other places throughout the code...these are placeholders?